x-pack/test/case_api_integration/security_and_spaces/tests/basic/index.ts

```diff
@@ -21,10 +21,14 @@ export default ({ loadTestFile, getService }: FtrProviderContext): void => {
     after(async () => {
       await deleteSpacesAndUsers(getService);
     });
-    // Common
-    loadTestFile(require.resolve('../common'));
 
     // Basic
     loadTestFile(require.resolve('./cases/push_case'));
+
+    // Common
+    loadTestFile(require.resolve('../common'));
+
+    // NOTE: These need to be at the end because they could delete the .kibana index and inadvertently remove the users and spaces
+    loadTestFile(require.resolve('../common/migrations'));
```

Comment:
I was getting occasional failures depending on the order that the tests were run in.

Reply:
👍🏾 Might be worth running this branch against the flaky test runner a few times